crypto/sleep.c

```diff
@@ -9,18 +9,23 @@
 
 #include <openssl/crypto.h>
 #include "internal/e_os.h"
+#include "internal/time.h"
+
+void ossl_nanosleep(uint64_t millis)
```

Comment:
Add `static` here - and condition it on the POSIX version as per @paulidale's suggestion.